### DISCUSSION TITLE
cant use Copilot Chat

### DISCUSSION THREAD

**Discussion by MadsFThaulow** [community/community#106945] [Copilot Conversations] [answered]
I have a copilot subscription and its working but when i go to the chat section it tells me to start a free trial but when i click it it ofc says i already have one, what do i do?
![image](https://github.com/community/community/assets/123249514/b6ee2ea1-2d1b-463f-8cbe-239ce5f2ef66)
![image](https://github.com/community/community/assets/123249514/993270c4-6aa6-45ec-a1be-374f371a9328)

**Comment by davecheney** [comment, marked as answer]
Duplicate of https://github.com/orgs/community/discussions/107550
